kubernetes-monitoring/mgmtagent_helm/values.yaml

```diff
@@ -1,12 +1,16 @@
 mgmtagent:
+   # Provide either installKeyValue or installKeyFile as an install key. If both provided then installKeyValue will take higher precedence.
+   
+   # Use the Base64 encoded value of Management Agent Install Key
+   installKeyValue:
    # Copy the downloaded Management Agent Install Key file under root helm directory as resources/input.rsp
-   installKey: resources/input.rsp
+   installKeyFile: resources/input.rsp
```

Comment:
We cannot change the name of the key as folks are already using this.  Can you change this back to "installKey"